### DISCUSSION TITLE
Command not working ... How to do.

### DISCUSSION THREAD

**Discussion by trytop** [vzhd1701/evernote-backup#13] [Q&A] [answered]
[root@s1 evernote]# **docker run --rm -t -v /workspace/docker/evernote:/tmp ghcr.io/vzhd1701/evernote-backup:latest**
Usage: evernote-backup [OPTIONS] COMMAND [ARGS]...

  Evernote backup & export

  On the first run, to export notes:
  evernote-backup init-db
  evernote-backup sync
  evernote-backup export output_dir/

Options:
  Verbosity: [mutually_exclusive]
    -q, --quiet                   Quiet mode, output only critical errors.
    -v, --verbose                 Verbose mode, output debug information.
  --version                       Show the version and exit.
  --help                          Show this message and exit.

Commands:
  init-db  Initialize storage & log in to Evernote.
  sync     Sync local database with Evernote, downloading all notes.
  export   Export all notes from local database into ENEX files.
  reauth   Refresh login to Evernote, run when token expires.
[root@s1 evernote]# **evernote-backup**
bash: evernote-backup: command not found...
[root@s1 evernote]#

**Comment by vzhd1701** [comment, marked as answer]
You have to use the whole docker run command and add `evernote-backup` specific commands after. Like this
`docker run --rm -t -v "$PWD":/tmp ghcr.io/vzhd1701/evernote-backup:latest init-db`
`docker run --rm -t -v "$PWD":/tmp ghcr.io/vzhd1701/evernote-backup:latest sync`
`docker run --rm -t -v "$PWD":/tmp ghcr.io/vzhd1701/evernote-backup:latest export output_dir/`

**Reply by trytop** [reply]
Thank you

**Comment by cheapnet** [comment]
Hello vzhd1701, I have tried to use this on my Synology NAS.  I managed to install docker on my NAS and when I run the docker run init-db command
`sudo docker run --rm -t -v "$PWD":/tmp ghcr.io/vzhd1701/evernote-backup:latest init-db`
I can see the container is running
It prompted me for "Username or Email" and after I entered my Evernote username, it stuck there without any further response.
I also tried to run the docker run sync command
`sudo docker run --rm -t -v "$PWD":/tmp ghcr.io/vzhd1701/evernote-backup:latest sync`
It prompted me for the password (without prompting me for the username.  Later found that it prompted me for the NAS password) and after I entered my Evernote password, it said "Sorry, try again."
Previously I can run the init-db and sync command using pip on my macbook and the same Evernote username and password without any issues.  It's just that my macbook is running out of space so I am keen to explore using my NAS for the Evernote backup.
Do you have any hint on these please? Do I have to use OAuth instead?

**Reply by vzhd1701** [reply]
Try running it with `-it` like this
`sudo docker run --rm -it -v "$PWD":/tmp ghcr.io/vzhd1701/evernote-backup:latest sync`
